data/protocols/1/LeagueDAO.json

```diff
@@ -13,7 +13,7 @@
     },
     "es": {
       "name": "League DAO",
-      "description": "League DAO is combining the best of DeFi protocols, play-to-earn gaming, and NFTs to create a web3 fantasy gaming reality where users can own and trade the players on their team and where on-chain scoring is based on real-world stats. Users can have a say in protocol governance with its native LEAG token."
+      "description": "League DAO combina lo mejor de los protocolos DeFi, juegos de jugar para ganar y NFT para crear una realidad de juegos de fantasía web3 donde los usuarios pueden poseer e intercambiar a los jugadores de su equipo y donde la puntuación en cadena se basa en estadísticas del mundo real. . Los usuarios pueden opinar sobre la gobernanza del protocolo con su token nativo LEAG."
```

Comment:
AS-IS: "League DAO combina lo mejor de los protocolos DeFi, juegos de jugar para ganar y NFT para crear una realidad de juegos de fantasía web3 donde los usuarios pueden poseer e intercambiar a los jugadores de su equipo y donde la puntuación en cadena se basa en estadísticas del mundo real. . Los usuarios pueden opinar sobre la gobernanza del protocolo con su token nativo LEAG."


TO-BE: "League DAO combina lo mejor de los protocolos DeFi, juegos de jugar para ganar y NFT para crear una realidad de juegos de fantasía web3 donde los usuarios pueden poseer e intercambiar a los jugadores de su equipo y donde la puntuación en cadena se basa en estadísticas del mundo real. Los usuarios pueden opinar sobre la gobernanza del protocolo con su token nativo LEAG."

Reply:
Done